include/Metabox/Posts/CategoryToTag.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+defined( 'ABSPATH' ) || exit; // Exit if accessed directly.
+
+/**
+ * Bulk Move Posts Category To Tag Metabox.
+ *
+ * @since 2.0.0
+ */
+class BM_Metabox_Posts_CategoryToTag extends BM_Metabox_Base {
+
+    protected function initialize() {
+        $this->meta_box_slug         = 'bm-posts-category-to-tag';
+        $this->messages['box_label'] = __( 'Move Posts from Category To Tag', 'bulk-move' );
+        $this->action                = 'move_category_to_tag';
+    }
+
+    public function render() {
+        $tags = $this->get_tags_or_fail();
+
+        if ( empty( $tags ) ) {
+            return;
+        }
+        ?>
+
+        <!-- Category To Tag Start-->
+        <h4><?php _e( 'On the left side, select the category whose post you want to move. In the right side select the tag to which you want the posts to be moved.', 'bulk-move' ); ?></h4>
+
+        <fieldset class="options">
+            <table class="optiontable">
+                <tr>
+                    <td scope="row" >
+	                    <?php
+		                    wp_dropdown_categories( array(
+			                    'name'             => 'smbm_mc_mapped_cat',
```

Comment:
The name `smbm_mc_mapped_cat` is not correct. Change it to `smbm_mct_cat`. Also make sure that the names that you use for unique across all modules. Otherwise it will result in name collusion.